### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by latenssi**

### CODE REVIEW

File: doc/REQUIREMENTS.md [hunk 1]
```diff
@@ -11,99 +11,102 @@ Note: "blind" is used in this document to mean “the contents of which are hidd
...
-* MUST expose the `Reveal` function on an owned Pack NFT, which:
-  * marks a Pack NFT as "Revealed"
-  * emits an on-chain event to be observed by the PDS with the Pack ID
-* MUST expose the `Withdraw` function on an owned Pack NFT, which:
-  * withdraws ALL Collectible NFTs from
+- MUST receive Pack NFTs after successful creation by the PDS
+- MUST expose the `Reveal` function on an owned Pack NFT, which:
+  - marks a Pack NFT as "Revealed"
+  - emits an on-chain event to be observed by the PDS with the Pack ID
+- MUST expose the `Withdraw` function on an owned Pack NFT, which:
```
Comment (aphelionz) [inline doc/REQUIREMENTS.md:111]: Just out of curiousity why `-` over `*`?

**Review by aphelionz — APPROVED** [review]
LGTM, just one bikeshedd-y question that isn't a show stopper at all

File: doc/REQUIREMENTS.md (hunk 1 above)
Comment (latenssi) [reply doc/REQUIREMENTS.md:111]: Didn't intend to do that! I will revert those changes 👍

Comment (latenssi) [reply doc/REQUIREMENTS.md:111]: Ah it is Prettier at it. Well we are going to face this someday so I'm rolling with it.

**Pull request merged**